api/specs/common/schemas/node-meta-v0.0.1-converted.yaml

```diff
@@ -291,3 +291,45 @@ properties:
             description: Units of the output value, if a physical quantity
             type: string
     additionalProperties: true
+  boot-options:
```

Comment:
one thing here. @pcrespov might confirm this as well.
I think in the JSon schema (which is sent to the frontend), the variable names are camelcase. you might need to set an alias cause I do not think we have such variables with ```-```.

Reply:
i confirm.

Reply:
It doe snot require a change here. Sorry, other files like `integration-version` are send the same way boot-options is encoded. I think it is correct